lldb/source/Commands/CommandObjectProcess.cpp

```diff
@@ -265,8 +265,6 @@ class CommandObjectProcessLaunch : public CommandObjectProcessLaunchOrAttach {
         process_sp->SyncIOHandler(0, std::chrono::seconds(2));
 
         llvm::StringRef data = stream.GetString();
```

Comment:
If we don't need `data` you could move it down to where it's used.

I'd say do `if (llvm::StringRef data = stream.GetString())...` but I don't think that will work because Stringref doesn't have `operator bool`.

Reply:
Done! Yes, as you expected, the second suggestion does not work.